.vscode/launch.json

```diff
@@ -0,0 +1,134 @@
+{
+    // see https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/debug-launch.md
+    "version": "0.2.0",
+    "configurations": [
+        // sclang target
+        {
+            "type": "lldb",
+            "request": "attach",
+            "name": "macOS: sclang attach (lldb)",
```

Comment:
Anyway to make this cross platform? It seems just the name is mac specific.

Reply:
macOS uses lldb and I think linux mostly relies on gcc/gdb(?) and then there is windows which is completely different - I think this could develop over time with contributions from people with different platforms, and maybe there is a better/more generic way of doing things, but I am not aware of one currently. this is one intention of the PR, to benefit of knowledge of other devs which will provide an improving setup for everyone over time.